app/models/volunteer_op.rb

```diff
@@ -34,4 +42,16 @@ def self.contains_description?(text)
   def self.contains_title?(text)
     arel_table[:title].matches(text)
   end
+  
+  def address_complete?
+    address.present? && postcode.present?
+  end 
+  
+  private
+  
+  def clear_lat_lng
+    return if address_complete?
+    self.longitude = nil
+    self.latitude = nil
+  end
```

Comment:
@ZmagoD I didn't suggest to destroy the method, just to add condition the the `after_confirmation` callback to make it available only to volunteer op from local source only like so: `after_validation :clear_lat_lng, if: "source == 'local'"`